### PR TITLE
Updated to add kafka capabilities for zeek

### PR DESCRIPTION
**Pull request by pcseanmckay**
Updated to add kafka vars to the env for the zeek live container.
Updated to add extra init containers to the zeek template and default values file.
Updated to add kafka vars in the default values file.

### CODE REVIEW

File: chart/templates/malcolm_configmaps.yaml [hunk 1]
```diff
@@ -358,6 +358,15 @@ metadata:
   name: zeek-env
 ---
 apiVersion: v1
+kind: ConfigMap
```
Comment (supcom234) [inline chart/templates/malcolm_configmaps.yaml:361]: Would not include the configmap at all if kafka enabled is set to false.

File: chart/templates/21-zeek-live.yml [hunk 2]
```diff
@@ -79,6 +79,8 @@ spec:
               name: upload-common-env
           - configMapRef:
               name: zeek-env
+          - configMapRef:
```
Comment (supcom234) [inline chart/templates/21-zeek-live.yml:82]: would not include this unless kafka is enabled.

File: chart/templates/malcolm_configmaps.yaml (hunk 1 above)
Comment (pcseanmckay) [reply chart/templates/malcolm_configmaps.yaml:361]: I will look into it.  However, having the extra envs doesn't do any harm.  They just won't be used.

File: chart/templates/21-zeek-live.yml (hunk 2 above)
Comment (pcseanmckay) [reply chart/templates/21-zeek-live.yml:83]: Done.

File: chart/templates/malcolm_configmaps.yaml (hunk 1 above)
Comment (pcseanmckay) [reply chart/templates/malcolm_configmaps.yaml:362]: Done.

**Review by supcom234 — APPROVED** [review]

**Pull request merged**